dref/utils.py

```diff
@@ -862,79 +460,54 @@ def extract_file(doc, created_by):
         pass
 
     try:
-        paragraph76 = document.paragraphs[75]._element.xpath('.//w:t')
-        media = []
-        for paragraph in paragraph76:
-            media.append(paragraph.text)
+        media = paragraphs[75] or []
         data['media_contact_title'] = media[3] if media[3] else None
         data['media_contact_email'] = media[5] if media[5] else None
         data['media_contact_phone_number'] = media[7] if media[7] else None
         data['media_contact_name'] = media[0] if media[0] else None
     except IndexError:
         pass
+
     # PlannedIntervention Table
     planned_intervention = []
     for i in range(8, 19):
-        table = document.tables[i]
-        try:
-            title = table.cell(0, 1)._tc.xpath('.//w:t')[0].text
-            budget = table.cell(0, 3)._tc.xpath('.//w:t')[1].text
-            targated_population = table.cell(1, 3)._tc.xpath('.//w:t')[0].text
-
-            indicators = []
-            title_1 = table.cell(3, 0)._tc.xpath('.//w:t')[0].text
-            target_1 = table.cell(3, 2)._tc.xpath('.//w:t')[0].text
-            indicator_1 = {
-                'title': title_1,
-                'target': target_1
-            }
-            indicators.append(indicator_1)
-        except(IndexError, ValueError):
-            pass
-        try:
-            title_2 = table.cell(4, 0)._tc.xpath('.//w:t')[0].text
-            target_2 = table.cell(4, 2)._tc.xpath('.//w:t')[0].text
-            indicator_2 = {
-                'title': title_2,
-                'target': target_2
-            }
-            indicators.append(indicator_2)
-        except(IndexError, ValueError):
-            pass
-        try:
-            title_3 = table.cell(5, 0)._tc.xpath('.//w:t')[0].text
-            target_3 = table.cell(5, 2)._tc.xpath('.//w:t')[0].text
-            indicator_3 = {
-                'title': title_3,
-                'target': target_3
-            }
-            indicators.append(indicator_3)
-        except(IndexError, ValueError):
-            pass
+        cells = get_table_cells(i)
+        title = cells(0, 1)
+        budget = cells(0, 3, 1)
+        targated_population = cells(1, 3)
```

Comment:
Please use targeted instead of targated. You could search for targated in the whole code (and fix it to targe...).